src/assets/eusde.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": ["asset"],
+  "symbol": "eusde",
+  "name": "Ethereal eUSDe",
+  "aliases": [
+    "eUSDe",
```

Comment:
Maybe we'll add `aEthEUSDE` and `stataEthEUSDE` for consistency? Because we did it for recent icons.